ecs-init/docker/docker.go

```diff
@@ -371,6 +371,10 @@ func (c *Client) getHostConfig(envVarsFromFiles map[string]string) *godocker.Hos
 		// bind mount instance config dir
 		config.InstanceConfigDirectory() + ":" + config.InstanceConfigDirectory(),
 	}
+	// only add update download dir if it is not the cache directory (the default setting)
+	if config.UpdateDownloadDir() != config.CacheDirectory() {
+		binds = append(binds, config.UpdateDownloadDir()+":"+config.UpdateDownloadDir())
```

Comment:
do we need to change the container path to be the same? if so, seems like customer has to specify the download dir to be an existing dir on agent's file system?